motr/m0kv/cmd_main.c

```diff
@@ -147,6 +159,23 @@ static int opts_get(struct params *par, int *argc, char ***argv)
 	if (is_str)
 		common_args++;
 
+	if (is_enf_meta)
+		common_args++;
+
+	if (pv != NULL) {
+		common_args += 2;
+		pv_token = strtok(pv, ":");
+		while (pv_token != NULL) {
+			if (pv_token_flag == 0) {
+				dix_pool_ver.f_container = strtoul(pv_token, NULL, 16);
+				pv_token_flag = true;
+			} else {
+				dix_pool_ver.f_key = strtoul(pv_token, NULL, 16);
+			}
+			pv_token = strtok(NULL, ":");	
+		}
```

Comment:
The format of this argument is FID?
Please use `client_fid_sscanf()` or `m0_fid_sscanf()` to get its value.

Reply:
> The format of this argument is FID?
Yes. Replaced with m0_fid_sscanf() function. Thanks.